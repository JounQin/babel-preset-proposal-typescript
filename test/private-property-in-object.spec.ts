import { Foo } from './private-property-in-object'
import { execute } from './helpers'

const proposal = 'private-property-in-object'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(new Foo().test({})).toEqual(false)
  expect(new Foo().test(new Foo())).toEqual(true)
})
