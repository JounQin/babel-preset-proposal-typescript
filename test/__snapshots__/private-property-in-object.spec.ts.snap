// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`private-property-in-object 1`] = `"Unexpected identifier"`;
